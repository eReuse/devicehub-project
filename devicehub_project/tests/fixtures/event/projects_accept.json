{
  "@type": "projects:Accept",
  "date": "2016-07-25T18:18:13",
  "description": "I accepted it because...",
  "byUser": "https://www.reutilitza.cat/users/manager",
  "project": "https://www.reutilitza.cat/projects/xyz",
  "url": "https://www.reutilitza.cat/projects/events/123456"
}