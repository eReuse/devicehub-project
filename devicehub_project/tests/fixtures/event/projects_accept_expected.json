{
  "@type": "projects:Accept",
  "date": "2016-07-25T18:18:13",
  "description": "I accepted it because...",
  "sameAs": "https://www.reutilitza.cat/projects/events/123456"
}